accession-commons-jpa/src/test/resources/features/modification.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Accession modification
+
+  Scenario Outline: I want to create a new patch of an existing object, in order to register improvements made to it
+  without losing access to the old information.
+    Given already accessioned <objects>
+    When user sends patch <input> for accession <accession>
+    Then user should receive version list for <accession> with 2 version
+    And patch must have versions increased
+    Examples:
+      | objects | input | accession    |
+      | A       | AA    | id-service-A |
+      | B       | BB    | id-service-B |
+      | A,B,C,E | AA    | id-service-A |
+
+  Scenario: I want to create a new patch of a non existing object
+    When user sends patch AA for accession id-service-A
+    Then user receives accession does not exist
+
+  Scenario Outline: I want to update the fields that identify uniquely an object.
+  without losing access to the old information.
+    Given already accessioned <objects>
+    When user updates <accession> patch <patch> with <input>
+    Then user should receive version list for <accession> with 1 version
+    And hash of version 1 should be <hash>
+    Examples:
+      | objects | input | accession    | patch | hash                                     |
+      | A       | AA    | id-service-A | 1     | 801C34269F74ED383FC97DE33604B8A905ADB635 |
```

Comment:
can we have Given hash as well , to show the hash has changed ?

Reply:
You mean the hash of the original object? Maybe I can stretch the language a bit, but the idea is that we are on the test  case of a set of updates and the information is already loaded so the user accion would be do the update not checking that the current element has hash X

Reply:
Ya i understand, i  thought the then was showing that the hash has changed , but the main use case is version not changed.